php-nginx/entrypoint.sh

```diff
@@ -38,6 +38,14 @@ if [ -f ${APP_NGINX_CONF} ]; then
     mv ${APP_NGINX_CONF} ${NGINX_DIR}
 fi
 
+# Configure memcached based session.
+if [ -n ${MEMCACHE_PORT_11211_TCP_ADDR} ] && [ -n ${MEMCACHE_PORT_11211_TCP_PORT} ]; then
```

Comment:
By checking both, it will prevent unintentional failure when either of them is missing. If we only check the combined string, there will be a chance of having incomplete configuration.

Or does it accept the save path like these?

```
; is it localhost?
session.save_path=:11211
```
or
```
; Is it the default port?
session.save_path=172.172.172.172: 
```